docs/instrumentation/tracing.md

```diff
@@ -0,0 +1,160 @@
+# Tracing
+
+`Tracer` is an entry point to the tracing capabilities and instrumentation. 
+It provides various functionalities for creating and managing spans, 
+extracting context from carriers, propagating context downstream, and more.
+
+### How to get the `Tracer`
+
+Currently, `otel4s` has a backend built on top of [OpenTelemetry Java][opentelemetry-java]. 
+Add the following configuration to the favorite build tool:
+
+@:select(build-tool)
+
+@:choice(sbt)
+
+Add settings to the `build.sbt`:
+
+```scala
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "otel4s-java" % "@VERSION@", // <1>
+  "io.opentelemetry" % "opentelemetry-exporter-otlp" % "@OPEN_TELEMETRY_VERSION@" % Runtime, // <2>
+  "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % "@OPEN_TELEMETRY_VERSION@" % Runtime // <3>
+)
+javaOptions += "-Dotel.java.global-autoconfigure.enabled=true" // <4>
+```
+
+@:choice(scala-cli)
+
+Add directives to the `*.scala` file:
+
+```scala
+//> using lib "org.typelevel::otel4s-java:@VERSION@" // <1>
+//> using lib "io.opentelemetry:opentelemetry-exporter-otlp:@OPEN_TELEMETRY_VERSION@" // <2>
+//> using lib "io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:@OPEN_TELEMETRY_VERSION@" // <3>
+//> using `java-opt` "-Dotel.java.global-autoconfigure.enabled=true" // <4>
+```
+
+@:@
+
+1. Add the `otel4s-java` library  
+2. Add an OpenTelemetry exporter. Without the exporter, the application will crash  
+3. Add an OpenTelemetry autoconfigure extension  
+4. Enable OpenTelemetry SDK [autoconfigure mode][opentelemetry-java-autoconfigure]  
+
+Once the build configuration is up-to-date, the `Tracer` can be created:
+
+```scala mdoc:silent
+import cats.effect.IO
+import org.typelevel.otel4s.trace.Tracer
+import org.typelevel.otel4s.java.OtelJava
+
+OtelJava.global.flatMap { otel4s =>
+  otel4s.tracerProvider.get("com.service").flatMap { implicit tracer: Tracer[IO] =>
+    val _ = tracer // use tracer here
+    ???
+  }
+}
+```
+
+### Creating a span
+
+You can use the `span` or `spanBuilder` API to create a new span. 
+
+The tracer automatically determines whether to create a child span or a root span based on the presence of a valid parent in the tracing context. 
+If a valid parent is available, the new span becomes a child of it; otherwise, it becomes a root span. 
```

Comment:
Semi-colon makes it a run-on sentence.

```suggestion
If a valid parent is available, the new span becomes a child of it.  Otherwise, it becomes a root span. 
```

Reply:
Thanks!